### PR TITLE
HRINT-356 Java Demos - FTS + Index Related Documents

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/demos.json [hunk 1]
```diff
@@ -20,7 +20,8 @@
         "demos": [
             "CreateRelatedDocuments",
             "LoadRelatedDocuments",
-            "QueryRelatedDocuments"
+            "QueryRelatedDocuments",
```
Comment (topalach) [inline DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/demos.json]: Please start with lower-case:
```
            "queryRelatedDocuments",
            "indexRelatedDocuments"
```

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/demos.json [hunk 2]
```diff
@@ -59,6 +60,16 @@
             "PagingQueryResults"
         ]
     },
+    {
+        "category": {
+            "slug": "text-search",
+            "directory": "textSearch"
+        },
+        "demos": [
+            "FTSWithStaticIndexSingleField",
```
Comment (topalach) [inline DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/demos.json]:
```
            "fTSWithStaticIndexSingleField",
            "fTSWithStaticIndexMultipleFields"
```

Comment (Danielle9897) [reply DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/demos.json]: done

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/demos.json (hunk 1 above)
Comment (Danielle9897) [reply DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/demos.json]: done

**Pull request merged**